web-admin/src/components/navigation/Breadcrumbs.svelte

```diff
@@ -29,7 +25,7 @@
   $: isOrganizationPage = $page.route.id === "/[organization]";
 
   $: projectName = $page.params.project;
-  $: project = createAdminServiceGetProject(orgName, projectName);
+  $: project = useProject(orgName, projectName);
```

Comment:
This is a good example why I think `useProject` should be renamed to `useProjectDeploymentStatus`. The name `useProject` on its own doesn't connote how it's different from the vanilla call to `createAdminServiceGetProject` that was here previously.

Reply:
Lets hold on this for now. I think there are a lot of these that we have punted on.